.gitignore

```diff
@@ -39,5 +39,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
-
+master.key
```

Comment:
this is already added

![image](https://user-images.githubusercontent.com/9253881/133995553-97d7e434-1a76-4c2d-88c6-ab62f2f9671e.png)

Reply:
removed another master.key